bf2fc6cc711aee1a0c2a/kas-fleet-manager
# chore: disable osd idp for kafka development environment by default

## Description
on dev environment we can rely on accessing the cluster by using kubeadmin

## Verification Steps

Run KAS Fleet Manager and add a data plane. Check that it ends up in `ready` state
Check that the following message appears in the logs when running kfm with a data plane:
```
`Configuration of data plane identity providers is disabled. Skipping configuring the identity provider for ClusterID 'xxxx'`
```



## Checklist (Definition of Done)
